features/step_definitions/login_steps.rb

```diff
@@ -1,14 +1,13 @@
 Given(/^the login page$/) do
-  #put your code here
-  pending
+  visit('/')
 end
 
 When(/^I log in with proper credentials$/) do
-  #put your code here
-  pending
+  fill_in('username', with: '1234')
+  fill_in('password', with: '1234')
+  click_button('Login')
 end
 
 Then(/^I should see the secret page$/) do
-  #put your code here
-  pending
+  expect(page).to have_content('This is the secret page')
```

Comment:
Note that you can also use Capybara's `current_path` method to figure out what page the user is on. However, it's up to you if you're rather test this by asserting what the page contains or by its path.